homeassistant/components/media_player/services.yaml

```diff
@@ -154,10 +154,14 @@ clear_playlist:
       description: Name(s) of entites to change source on
       example: 'media_player.living_room_chromecast'
 
-sonos_group_players:
-  description: Send Sonos media player the command for grouping all players into one (party mode).
+sonos_join:
+  description: Group player toggether.
 
   fields:
+    master:
+      description: this player is the coordinator of the group. Platform dependent.
```

Comment:
description should be `Entity ID of the player that should become the coordinator of the group.`